Automattic/jetpack
# Related Posts: display different configuration link based on theme

Fixes #31823

## Proposed changes:

If the site owner uses a block theme on their site, let's invite them to use template parts to customize related posts, instead of directing them to the customizer.

> **Note**
> I'm not sure the wording is nice enough, I'd be happy to have an opinion on that.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start on a site connected to WordPress.com.
* Go to Jetpack > Settings > Traffic
* Enable the Related Posts feature
* Check that the configuration link at the bottom of the card works, and that the wording is accurate.
* Go to Appearance > Themes, and switch to a different theme ; if you used a classic theme, switch to a block theme ; if you used a block theme switch to a classic theme
* Go back to Jetpack > Settings > Traffic
* Check the new link
